packages/syft/src/syft/service/worker/utils.py

```diff
@@ -374,28 +391,33 @@ def run_workers_in_kubernetes(
     worker_count: int,
     pool_name: str,
     queue_port: int,
-    start_idx=0,
+    start_idx: int = 0,
     debug: bool = False,
     reg_username: Optional[str] = None,
     reg_password: Optional[str] = None,
     reg_url: Optional[str] = None,
-    **kwargs,
+    **kwargs: Dict[str, Any],
```

Comment:
```suggestion
    **kwargs: Any,
```

See https://github.com/OpenMined/PySyft/pull/8464#discussion_r1481464164

Reply:
Fixed. Thanks @kiendang